modules/tinymce/CHANGELOG.md

```diff
@@ -21,6 +22,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A double bottom border appeared on inline mode editor for the `tinymce-5` skin. #TINY-9108
 - The editor header showed up even with no menubar and toolbar configured. #TINY-8819
 
+### Changed
```

Comment:
Forgot to leave a comment... this should be above `Fixed` and I don't think this makes sense as `Changed`. Maybe the first one, but the second one is definitely `Improved` or `Added`.